mobile/lib/models/enum_constants.dart

```diff
@@ -213,6 +214,83 @@ enum AirQuality {
     maximumValue: 500,
   );
 
+  String getSearchNearbyLocationsText(BuildContext context) {
+    switch (this) {
+      case AirQuality.good:
+        return AppLocalizations.of(context)!.goodQualityAirAroundYou;
+      case AirQuality.moderate:
+        return AppLocalizations.of(context)!.moderateQualityAirAroundYou;
+      case AirQuality.ufsgs:
+        return AppLocalizations.of(context)!
+            .nearbyLocationsWithAirQualityUnhealthyForSensitiveGroups;
```

Comment:
AppLocalizations.of(context)!
            .nearby........... it used by flutter to replace with the right languages(locale)